skyportal/handlers/api/source.py

```diff
@@ -274,13 +290,20 @@ def get(self, obj_id=None):
             )
         if has_tns_name in ['true', True]:
             q = q.filter(Obj.altdata['tns']['name'].isnot(None))
-
+        if group_id:
+            if not group_id.isdigit() or int(group_id) not in [
```

Comment:
We should check separately whether it's not a digit, and return a more specific error message in that case.

Reply:
yes, done